src/name_service/storage/mongo.md

```diff
@@ -0,0 +1,42 @@
+# mongoDB
+
+Microservice using [Clojure Monger](http://clojuremongodb.info/) to connect with MongoDB.
+
+## Getting Started
+
+1. Install and start MongoDB. [How to install/run.](https://docs.mongodb.com/manual/administration/install-community/)
```

Comment:
Don't point to manual install, if you want to suggest how to run it, use Docker.